app/src/main/res/layout/fragment_claim_helium_reset.xml

```diff
@@ -20,14 +20,13 @@
 
             <com.google.android.material.textview.MaterialTextView
                 android:id="@+id/resetTitle"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:text="@string/reset_station"
+                android:text="@string/reboot_your_device"
```

Comment:
This was by mistake
It should be "Reset your device", check the updated designs